sdk/src/Core/Amazon.Runtime/Internal/Util/StringUtils.cs

```diff
@@ -384,17 +384,11 @@ public static string FromValueTypeList<T>(List<T> values) where T : struct
             }
             // See https://datatracker.ietf.org/doc/html/rfc7231.html#section-7.1.1.1
             // FromDateTimeToRFC822 is compatible with IMF-fixdate
-#if NET35
```

Comment:
There are a few places where we use `BCL35`: https://github.com/search?q=repo%3Aaws%2Faws-sdk-net%20BCL35&type=code

Should we remove them too?

Reply:
Yes, I didn't search for that string but that is a good find. I will clean that up.

Reply:
Latest code cleaned up BCL35 and also AWS_ASYNC_API (manual changes only in this PR). DryRuns in progress.